param/parameterized.py

```diff
@@ -1215,6 +1238,7 @@ def set_param(self_, *args,**kwargs):
         positional arguments, but the keyword interface is preferred
         because it is more compact and can set multiple values.
         """
+        BATCH_WATCH = self_.self_or_cls.param._BATCH_WATCH
```

Comment:
Why the scary capitals?

Reply:
Just following the style of the original variable name, not sure it has to be capitalized but if we decide to change I would change it everywhere.

Reply:
As you wish.